fenix/app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -337,6 +341,10 @@ class FirefoxColors(
     var layer3 by mutableStateOf(layer3)
         private set
 
+    // New Search
```

Comment:
I suspect this is probably `// Search` now. However, I am really hesitant on adding a comment here. The other comments that you see here were originally defined in the figma stickersheet many versions ago. I would rather we didn't have a comment over coming up with a comment that the design system team didn't define for us to avoid confusion down the line. 

If we were to have a comment, then I think we should stick with `// Search`. I suspect layer3 doesn't really refer to search anymore given its multiple usage everywhere https://searchfox.org/mozilla-mobile/search?q=layer3&path=, but that needs to be verified -- so no action to be taken there.

Reply:
Yeah I don't see any comment associated from where I am [looking](https://www.figma.com/file/pEyGeE4KV5ytYHeXMfLcEr/Mobile-Styles?type=design&node-id=5276-17855&mode=design&t=hH19SxWSrG6c6HbW-4). So does the comment "New Search" need to be removed?